solidity/dashboard/src/services/token-grants.service.js

```diff
@@ -77,18 +103,22 @@ const getGrantDetails = async (
     "unlockedAmount",
     grantId
   )
-  let readyToRelease = "0"
+  let withdrawableAmountGrantOnly = "0"
   try {
```

Comment:
wonder if we need try catch block here. In other places we do not use it. Did we add it here for a specific reason?

Reply:
We probably don't need this anymore, I'll double-check. We added `try catch` block to catch the `SafeMath underflow` bug on the smart contract side that was addressed with https://github.com/keep-network/keep-core/pull/1464.